imagery/i.ortho.photo/i.ortho.photo/menu.c

```diff
@@ -96,26 +98,30 @@ int main(int argc, char **argv)
     moduletorun = ortho_opt->answer;
     /* run the program chosen */
     if (strcmp(moduletorun, "g.gui.photo2image") == 0) {
-        strcpy(tosystem, "g.gui.photo2image");
+        strncpy(tosystem, "g.gui.photo2image", BUFFER_SIZE - 1);
+        tosystem[BUFFER_SIZE - 1] = '\0';
         return system((const char *)tosystem);
     }
     else if (strcmp(moduletorun, "g.gui.image2target") == 0) {
-        strcpy(tosystem, "g.gui.image2target");
+        strncpy(tosystem, "g.gui.image2target", BUFFER_SIZE - 1);
+        tosystem[BUFFER_SIZE - 1] = '\0';
         return system((const char *)tosystem);
     }
     else {
         if (strcmp(moduletorun, "i.group") == 0)
-            strcpy(tosystem, "i.group --ui group=");
+            strncpy(tosystem, "i.group --ui group=", BUFFER_SIZE - 1);
```

Comment:
```suggestion
            strncpy(tosystem, "i.group --ui group=", BUFFER_SIZE - 1);
            tosystem[BUFFER_SIZE - 1] = '\0';
```

But, with https://github.com/OSGeo/grass/pull/4101, will be simply: 

`strlcpy(tosystem, "i.group --ui group=", BUFFER_SIZE);`

Reply:
ok i will wait then till #4101 gets merged.
I will then use this G_strlcpy

Reply:
#4101 was merged

Reply:
i will do the changes

Reply:
changes done please review it